api/helpers/urlToBlob.js

```diff
@@ -0,0 +1,32 @@
+const https =require("https");
+
+module.exports = {
+    downloadFileFromUrl
+};
+
+async function downloadFileFromUrl(baseUrl){
+    let uploadedFile = undefined;
+  
+    return new Promise((resolve, reject) => {
```

Comment:
You might be able to use `axios` here. Here's an [example](https://github.com/cboard-org/cboard/blob/master/src/components/Settings/Export/Export.helpers.js#L94).